src/main/java/uk/gov/hmcts/reform/sscs/helpers/CaseFlagMappingServiceHelper.java

```diff
@@ -0,0 +1,106 @@
+package uk.gov.hmcts.reform.sscs.helpers;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+@SuppressWarnings("PMD")
+public class CaseFlagMappingServiceHelper {
+
+    private final String[] mappingValues =
```

Comment:
Should we make these enums instead or add as descriptions to the existing `FlagCode` enums?

Reply:
I will look into this.